src/Gui/Stylesheets/Dark-blue.qss

```diff
@@ -2114,7 +2114,7 @@ SELECTORTOOLBAR widget (3rd party plugin)
 ==================================================================================================*/
 QToolBar:horizontal#Selector,
 QToolBar:vertical#Selector {
-    background-color: rgba(0,0,0,120);
+    /* background-color: rgba(0,0,0,120); */

```

Comment:
why this commit? IIRC this was deliberately added to the stylesheet so selector toolbar gets a different color from the rest of the toolbars. From my understanding this would make it look the same as all others.